DistributedLock.Tests/Infrastructure/Redis/TestingRedisSynchronizationStrategy.cs

```diff
@@ -22,6 +22,15 @@ public void SetOptions(Action<RedisDistributedSynchronizationOptionsBuilder>? op
             this._options = options;
         }
 
+        public override string? GetLockPrefix()

```

Comment:
Why is this needed?

Reply:
The need to run RunLockTaker / DistributedLockTaker with global lock name (prefix + local name) in order to pass cross process tests.